WalletWasabi.Fluent/Behaviors/RandomizedWorldPointsBehavior.cs

```diff
@@ -110,7 +110,7 @@ private void RunAnimation(CancellationToken cancellationToken)
 			cancellationToken);
 	}
 
-	private async Task AnimateCityMarkerAsync(IControl target, Point point, CancellationToken cancellationToken)
+	private async Task AnimateCityMarkerAsync(Control target, Point point, CancellationToken cancellationToken)
```

Comment:
We will need more changes like this for the latest avalonia 11 nightly. Interface removal would be a PITA but this reduces the pain later :)

Reply:
We also need behaviors and treedatagrid updates or we will get exception for missing types.